tests/orion/models/test_work_queues.py

```diff
@@ -270,16 +272,23 @@ async def scheduled_flow_runs(self, session, deployment, work_queue, work_queue_
 
     @pytest.fixture
     async def running_flow_runs(self, session, deployment, work_queue, work_queue_2):
-        for i in range(3):
+        for i in range(self.running_flow_count):
```

Comment:
It seems like this could just use a list of state types instead? The numerical index doesn't seem very clear.

Reply:
:+1: Works for me.